platform-ajax/src/test/java/com/softicar/platform/ajax/input/auto/complete/AbstractAjaxAutoCompleteTest.java

```diff
@@ -144,23 +144,25 @@ private void assertContainsText(String string, String subString) {
 
 	public static enum Indicator implements IAjaxSeleniumTestDomAutoCompleteInputIndicator {
 
+		// TODO remove COMMITTING and LOADING
+		// TODO use DomAutoCompleteIndicatorType directly
 		COMMITTING("committing", "AutoCompleteIndicatorCommitting"),
 		LOADING("loading", "AutoCompleteIndicatorLoading"),
-		NOT_OKAY("not-okay", "AutoCompleteIndicatorNotOkay"),
-		VALUE_AMBIGUOUS("value-ambiguous", "AutoCompleteIndicatorValueAmbiguous"),
-		VALUE_ILLEGAL("value-illegal", "AutoCompleteIndicatorValueIllegal"),
-		VALUE_MISSING("value-missing", "AutoCompleteIndicatorValueMissing"),
-		VALUE_VALID("value-valid", "AutoCompleteIndicatorValueValid"),
+		NOT_OKAY("not-okay", "DomAutoCompleteIndicatorNotOkay"),
```

Comment:
second arguments in this enum should use enumerated CSS classes, instead of plain strings

Reply:
replaced class with DomAutoCompleteIndicatorType